sagemaker-clarify/clarify-explainability-inference-pipelines/credit_risk_explainability_inference_pipelines.ipynb

```diff
@@ -0,0 +1,1482 @@
+{
```

Comment:
<p>Preferred to add code to download from S3, so customers don't need to manually download and upload the dataset. Are you using the SouthGermanCredit.asc file from that link? Please use our hosted version<code>s3://sagemaker-sample-files/datasets/tabular/uci_statlog_german_credit_data/SouthGermanCredit.asc</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2866/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
fixed